index.js

```diff
@@ -240,7 +240,7 @@ NYC.prototype.walkAllFiles = function (dir, visitor) {
 }
 
 NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
-  var instrument = this.exclude.shouldInstrument(filename, relFile)
+  var instrument = filename.indexOf('mono/medium2') >= 0 || this.exclude.shouldInstrument(filename, relFile)
```

Comment:
could you make this injectable via the config object? i don't want to have to change the library every time we need to add a new directory. We have JS tests outside of medium2.